IM_calculation/scripts/calculate_ims.py

```diff
@@ -199,8 +199,8 @@ def main():
     if "pSA" in im:
         im_options["pSA"] = calc.validate_period(args.period, args.extended_period)
 
-    if "IESD" in im:
-        im_options["IESD"] = calc.validate_period(args.period, args.extended_period)
+    if "SDI" in im:
+        im_options["SDI"] = calc.validate_period(args.period, args.extended_period)
```

Comment:
Can possibly merge the two calls to validate_period rather than having duplicate / copied code.

Reply:
Done